packages/docusaurus-utils/src/markdownParser.ts

```diff
@@ -51,14 +51,13 @@ export function createExcerpt(fileString: string): string | undefined {
 
     // Skip code block line.
     if (fileLine.trim().startsWith('```')) {
+      const codeFence = fileLine.trim().match(/^`+/)![0]!;
```

Comment:
🤔 I have no idea what is going on here and that looks like a risky change. Are tests covering this?

Reply:
Yes, we do have tests for this. We also have edge cases for this on our own website, so if our website builds, it's good!